dl4j-examples/src/main/java/org/deeplearning4j/examples/modelimport/tensorflow/LoadTensorFlowMNISTMLP.java

```
@@ -0,0 +1,94 @@
+ * Currently a subset of lower level ops are supported i.e higher level abstractions (like tf.layers.dense) are not tested/supported
+ * Expanded set of ops will be supported in the upcoming release
+ * TensorFlow graphs need to be "frozen" and saved to a protobuf before import.
+ * "freezing" refers to the process of converting all variables in the graph to constant and pruning the graph of all nodes/ops not necessary for inference.
+ * The TensorFlow documentation on freezing graphs: https://www.tensorflow.org/extend/tool_developers/#freezing
+ *
+ * @author susaneraly
+ */
+public class LoadTensorFlowMNISTMLP {
+
+    //Python code for this can be found in resources/import/tensorflow under generate_model.py and freeze_model_after.py
+    //Input node/Placeholder in this graph is names "input"
+    //Output node/op in this graph is names "output"
+    public final static String BASE_DIR = "import/tensorflow";
+
+    public static void main(String[] args) throws Exception {
+        final String FROZEN_MLP = new ClassPathResource(BASE_DIR + "/frozen_model.pb").getFile().getPath();
+
+        //Load placeholder inputs and corresponding predictions generated from tensorflow
+        Map<String, INDArray> inputsPredictions = readPlaceholdersAndPredictions();
+
+        //Load the graph into samediff
+        val graph = TFGraphMapper.getInstance().importGraph(new File(FROZEN_MLP));
+        //libnd4j executor
+        //running with input_a array expecting to get prediction_a
+        graph.associateArrayWithVariable(inputsPredictions.get("input_a"), graph.variableMap().get("input"));
+        val executioner = new NativeGraphExecutioner();
+        val results = executioner.executeGraph(graph); //returns an array of the outputs
+        INDArray libnd4jPred = ((INDArray[]) results)[0];
+        System.out.println("LIBND4J exec prediction for input_a:\n"+libnd4jPred);
+        if (libnd4jPred.equals(inputsPredictions.get("prediction_a"))) {
```

Nitpick: we should throw an error if the outputs do not match.